biggus/tests/unit/test_Array.py

```diff
@@ -240,6 +235,20 @@ def test_other_no_good(self):
             a // None
 
 
+class Test___div__(unittest.TestCase, AssertElementwiseMixin):
+    def test_other_array(self):
+        a = FakeArray([2, 4])
+        b = FakeArray([2, 4])
+        r = operator.div(a, b)
```

Comment:
Cool - I've put that in, but had change one check as `a.__div__(None)` does not raise a `TypeError` but instead returns `NotImplemented`